update incompatibilities section (#2420)
php/frankenphp: docs/known-issues.md

| ----------------------------------------------------------------------------------------------------------- | --------------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| [imap](https://www.php.net/manual/imap.installation.php)                                                 | Not thread-safe | [javanile/php-imap2](https://github.com/javanile/php-imap2), [webklex/php-imap](https://github.com/Webklex/php-imap), [ImapEngine](https://github.com/DirectoryTree/ImapEngine) |
| [newrelic](https://docs.newrelic.com/docs/apm/agents/php-agent/getting-started/introduction-new-relic-php/) | Not thread-safe | -                                                                                                                                                                               |

> ## Buggy PHP extensions

Author: ```suggestion
## PHP extensions with limited support
```

> The following extensions have known bugs and unexpected behaviors when used with FrankenPHP:

Author: ```suggestion
The following extensions are in beta or have known issues when used with FrankenPHP:
```

| Name                                  | Problem                                                                                                                                         |
|---------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
| [datadog](https://github.com/DataDog) | Experiences instability when profiling FrankenPHP. This bug is [being tracked by DataDog](https://github.com/DataDog/dd-trace-php/issues/3729). |

> | [blackfire](https://blackfire.io/) | [FrankenPHP support is in beta and not yet feature-complete](https://docs.blackfire.io/php/integrations/frankenphp).                            |

Reviewer: To me it shouldn't be marked as "buggy". It works perfectly well in normal mode.

Author: both of them? I've seen crashes for both not too recently

Reviewer: Blackfire. But if they both crash, then let's keep them.

Author: Blackfire is using datadog for continuous profiling (ZTS) under the hood, so it might be related to that. Perhaps we can rename the header from "buggy" to something else, but I'm not sure to what.
